exasol/ds/sandbox/runtime/ansible/roles/jupyter/files/notebook/cloud/02_s3_vs_reuters.ipynb

```diff
@@ -0,0 +1,287 @@
+{
```

Comment:
<p>maybe we should also showcase a join between topics and news, perhaps by the most frequent topics since some data</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/exasol/ai-lab/pull/355/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>